plugins/kotlin/j2k/shared/tests/testData/newJ2k/field/initializedBothWithInitializerAndConstructor.kt

```diff
@@ -1,5 +1,6 @@
 class TestFieldInitializer(string: String) {
     var string: String = ""
+        private set
```

Comment:
I believe this test case was initially wrong. It seems like `private set` needs to be there else `string` could be set from outside the class